bittensor/_dataset/__init__.py

```diff
@@ -161,10 +191,10 @@ def add_defaults(cls, defaults):
         """
         defaults.dataset = bittensor.Config()
         defaults.dataset.batch_size = os.getenv('BT_DATASET_BATCH_SIZE') if os.getenv('BT_DATASET_BATCH_SIZE') != None else 10
-        defaults.dataset.block_size = os.getenv('BT_DATASET_BLOCK_SIZE') if os.getenv('BT_DATASET_BLOCK_SIZE') != None else 20
+        defaults.dataset.block_size = 1500
```

Comment:
should still follow the convention of allowing an environment variable. Also, we need one for the new sequence length yeah?